HACKING.md

```diff
@@ -2,13 +2,35 @@
 
 To work in the framework itself you will need Python >= 3.8. Linting, testing,
 and docs automation is performed using
-[`tox`](https://tox.readthedocs.io/en/latest/), which you should install.
-For improved performance on the tests, ensure that you have PyYAML
-installed with the correct extensions:
+[`tox`](https://tox.readthedocs.io/en/latest/).
+
+First, make sure to install [uv](https://docs.astral.sh/uv/), for example via:
+
+```sh
+sudo snap install astral-uv --classic
+```
+
+Then install `tox` with extensions, as well as a range of Python versions:
+
+```sh
+uv python install 3.8 3.10 3.11 3.12
+uv tool install tox --with tox-uv
+uv tool update-shell
+```
+
+You can validate that you have a working installation by running:
+```sh
+tox --version
+4.26.0 from /home/<your-user>/.local/share/uv/tools/tox/lib/python3.13/site-packages/tox/__init__.py
+registered plugins:
+    tox-uv-1.26.0 at /home/<your-user>/.local/share/uv/tools/tox/lib/python3.13/site-packages/tox_uv/plugin.py with uv==0.7.12
+```
+
+For improved performance on the tests, install the library that allows
+PyYAML to use C speedups:
 
 ```sh
 apt-get install libyaml-dev
```

Comment:
This command needs a `sudo` prefix, right? Should we update it while we're here.